plugins/sfRadPlugin/lib/sfRadPlugin.class.php

```diff
@@ -50,45 +50,7 @@ public function __toString()
         return $string;
     }
 
-    public function __get($name)
```

Comment:
Can we remove the `ArrayAccess` implementation in this class after this changes?

Reply:
Done, removed the offset methods and the `ArrayAccess` implementation. We don't seem to be calling the offset methods anywhere and this doesn't seem to have affected the index or the edit pages.